client/state/happychat/selectors.js

```diff
@@ -1,7 +1,7 @@
 /**
  * External dependencies
  */
-import { map, head, get } from 'lodash';
+import { get, head, map } from 'lodash';
```

Comment:
I think this came up during conflict resolution when rebasing on master. You and I both made the same change in separate PRs (replacing the three separate import lines) but I hadn't alphabetized the imports. So when resolving the conflict I went with this PR's alphabetized version.